webui/test/e2e/common/uploadFile.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from '@playwright/test';
+import { RepositoriesPage } from "../poms/repositoriesPage";
+import { RepositoryPage } from "../poms/repositoryPage";
+import fs from "fs";
+import path from "path";
+
+const TEST_REPO_NAME = "test-upload";
+const FILE_NAME = "test-upload.txt";
+
+
+test.describe("Upload File", () => {
+    test.describe.configure({mode: "serial"});
+		test("Create repo", async ({page}) => {
+			const repositoriesPage = new RepositoriesPage(page);
+			await repositoriesPage.goto();
+			await repositoriesPage.createRepository(TEST_REPO_NAME, true);
+		});
```

Comment:
The create repository should be a step in the same test, the upload file test shouldn't test the create repository step